semantic.cabal

```diff
@@ -67,7 +67,6 @@ common dependencies
                      , semantic-source ^>= 0.0
                      , semilattices ^>= 0.0.0.3
                      , streaming ^>= 0.2.2.0
-                     , streaming-bytestring ^>= 0.1.6
```

Comment:
We don’t use this anywhere other than the parse-examples and semantic-python tests, so it can be dropped from the common one.